src/app/components/buscadorTurnosPrestaciones/turnos-prestaciones.component.ts

```diff
@@ -106,11 +106,24 @@ export class TurnosPrestacionesComponent implements OnInit {
             estado: '',
             estadoFacturacion: '',
         };
+
+        if (this.auth.profesional) {
+            this.serviceProfesional.get({ id: this.auth.profesional }).subscribe(rta => {
+                this.profesional = rta[0];
+                params.idProfesional = this.profesional.id;
+                this.selectProfesional = true;
```

Comment:
Faltaría agregar this.parametros['idProfesional'] = this.profesional.id , porque al filtrar te trae prestaciones de otros profesionales aunque este en disabled el filtro de "Equipo de salud"

Reply:
Modificado